### PR TITLE
Fix the integration test to use config properly

### PR DESCRIPTION
**Pull request by varjoranta**
@jstck @steffengr

### CODE REVIEW

**Comment by varjoranta** [comment]
This should be ready for merge now. We should be able to make our first proper release after this one.

File: README.md [hunk 1]
```diff
@@ -346,29 +346,51 @@ an ephemeral cluster as long as the cluster is running.
 * [gsutil](https://cloud.google.com/storage/docs/gsutil) authenticated with the service account
 
 ### Integration Test Configuration
+
 In order to run integration tests, basic configuration needs to be provided during the build process.
-Create a spydra_conf.json file similar to the one below and reference it during the maven invocation.
+Create a file with name *integration-test-config.json* similar to the one below and reference
+it during the maven invocation.
 
 ```$xslt
```
Comment (Xeago) [inline README.md]: xslt doesn't look right xD

File: .travis.yml [hunk 2]
```diff
@@ -30,11 +31,14 @@ branches:
   only:
   - master
 
+# Skipping the install step or otherwise it would trigger enforcer plugin and fail.
+install: true
```
Comment (steffengr) [inline .travis.yml]: Why do we want to skip this and if so shouldn't we remove the plugin then?

File: .travis.yml [hunk 3]
```diff
@@ -12,13 +12,14 @@ env:
...
     - secure: GkVJrZ1PQn8wT+wL7zSjjvDWKvooPyP+ESkkGaqAxlfl+miFaB7YPWPL3K9JpvodZ3uh+7W2hd1WuqSlTQo7sKReaH3hdfYWO43JgcLkVf7dNiVysEOuNFesRawvchKHYy5+QdHfwX3gqBmFJWqVXsBL3v4NVQZaEmhSAxMoSjwJsAamArPG/3DHlt0daTXC9yeraG3cF5GiBeuYrbSzvYMF1z0Z5te9E7F68f7xZCjGFq2cgYQbLyxzErZFXJZcxrMoz6i/HNcChEVQmqqLqNt4rykyDVnjy4IhWw45eujBnBvs0qwBVGIjaarl0ZmQh8dV+DbOQIA3wxXFQVhJdMeu2tai+MHRWYHM+Q3vfrO5Tv3CpWLaKroJeRRNyHLeTtmAUT4JE5D/kpUS71BuWF1ibWVWBKGNNoKZzz2DwqAf4tj1D1DQL+qt7PqXHgqOL3pTT3FuXIpR07TqMlmlnsH3DCyEwTD4f43KMlrj93G+AowVYTP/jlCXAxoO4I4IOJeqpWiEg7LGqdzBYkI6j3po7vASvJzNVUWIn9glf+lljLrV7qH8AMaD4S4n+2XvxMrBodDx8QhcvMCDIDPZOQ13xrhGCR8veu4D3qeA3dtL4RQ/0PAg7J/Ar9RgUdRDXggNDMmN+uCHJY4t5dmhp7CENTsA2Y0jNawgM30gccM=
 
 before_install:
   # Decrypt the credentials for accessing GCP project (only non pull-requests)
-  - openssl aes-256-cbc -K $encrypted_e1de2a468852_key -iv $encrypted_e1de2a468852_iv -in secrets/spydra_foss_ci.json.enc -out ${GOOGLE_APPLICATION_CREDENTIALS} -d
+  - openssl aes-256-cbc -K $encrypted_e1de2a468852_key -iv $encrypted_e1de2a468852_iv -in travis_files/spydra_foss_ci.json.enc -out ${GOOGLE_APPLICATION_CREDENTIALS} -d
```
Comment (Xeago) [inline .travis.yml]: use $TEST_CONFIG_FOLDER.

File: .travis.yml (hunk 2 above)
Comment (varjoranta) [inline .travis.yml]: I can update the comment to be more precise. The main reason is that we actually do not need the install step at all, ac the script step will do the same, i.e. with install step we would just run the maven twice.

**Pull request merged**